Doc/library/dataclasses.rst

```diff
@@ -185,7 +185,10 @@ Module contents
    - ``slots``: If true (the default is ``False``), :attr:`~object.__slots__` attribute
      will be generated and new class will be returned instead of the original one.
      If :attr:`~object.__slots__` is already defined in the class, then :exc:`TypeError`
-     is raised.
+     is raised. Calling no-arg ``super()`` in dataclasses using ``slots=True`` will result in
+     the following Exception being raised:
+     ``TypeError: super(type, obj): obj must be an instance or subtype of type``.
+     The two-arg ``super()`` is a valid workaround. See :issue:`46404` for full details.
```

Comment:
We should try to capture everything relevant in the documentation -- what is relevant to the reader from the linked issue that isn't captured in your summary?

A

Reply:
I _think_ these two points are it. What to look for and how to workaround.